kge/config-default.yaml

```diff
@@ -220,6 +220,9 @@ KvsAll:
   # ( https://github.com/TimDettmers/ConvE/blob/853d7010a4b569c3d24c9e6eeaf9dc3535d78832/main.py#L156) with a default value of 0.1.
   label_smoothing: 0.0
 
+  # If set, training also runs on subject object (so) pairs additionally to sp and op pairs
```

Comment:
Please rebase to master and then change options to: 
```
queries:
   sp_: True
   s_o: False
   _po: True
```
I.e., each of the KvsAll prediction tasks can be enabled/disabled individually.